Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -1091,6 +1090,8 @@ public void setExperimentalMarkerScaleFactor(double experimentalMarkerScaleFacto
                         experimentalMarkerScaleFactor*DEFAULT_MARKER_SIZE);  
             }
         }
+        // update cached experimentalMarkerGeometryJson
+         experimentalMarkerGeometryJson.put("radius", DEFAULT_MARKER_SIZE*experimentalMarkerScaleFactor);
```

Comment:
See previous comment.

Reply:
This line is actually the meat of the fix, which is that when user updates the scale factor, the cached marker geometry wasn't kept in sync, now it does. Hope this explains.